### DISCUSSION TITLE
Easy switch from "Files changed" to "Conversation" tab

### DISCUSSION THREAD

**Discussion by AurelienGasser** [community/community#32768] [Pull Requests]
In my day-to-day workflow, I encounter a usability issue with big PRs (lots of files, lots of discussion). The typical scenario is I am in the middle of reviewing files, and I want to read the PR description again. So I:

1. Click on the PR title
2. And then click on the "Conversation" tab

This creates 2 problems:

- The click on the title in step 1 scrolls back to the top of the page, so I lose my scroll position. If I want to continue reviewing the files I have to figure out how far down the page I was + scroll down there. This takes several seconds for big PRs. Note that control-click on the title doesn't work (it doesn't open a new browser tab)

- Step 2 can take a good 5-10 secs: the "Conversation" tab is pretty heavy for big PRs.

I do this a lot throughout the day so it adds up to a lot of wasted time.

There doesn't seem to be a keyboard shortcut to go from the "Files changed" to the "Discussion" tab.

There are 2 alternative workflows, but both have issues

1. Always open the "Files changed" tab in a new browser tab. The problem with this approach is that I need to open (and keep open) 2 tabs per PR.
2. Review PRs in the in-browser VSCode by using the `.` shortcut. The issue is it takes 15 secs for me to get to the description this way (with the browser cache warmed up)

Is there a better workflow?

![image](https://user-images.githubusercontent.com/168382/189651893-34f063e9-a6b9-4fa8-a4d9-d77aaaecf89a.png)

**Comment by tonglil** [comment]
What I want is to be able to middle-click on the PR title and open the `conversation` tab in a new tab.

Right now it's fairly useless as it just takes me to the top of the page, which means middle-clicking it opens a new tab of the `files` tab, and it takes another click to get to the `conversation` tab.
